src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -10,6 +10,7 @@ import FeedIcon from '../../../assets/images/feed-icon.png';
 import dimen from '../../../utils/dimen';
 import {BaseChannelItemTypeProps} from '../../../../types/component/AnonymousChat/BaseChannelItem.types';
 import {colors} from '../../../utils/colors';
+import useProfileHook from '../../../hooks/core/profile/useProfileHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useProfileHook` is imported but it's not clear from the provided context whether this hook is defined correctly and handles all edge cases. Please ensure that this hook is thoroughly tested and handles all possible scenarios.
```diff
+ import useProfileHook from '../../../hooks/core/profile/useProfileHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->